### DISCUSSION TITLE
Build .NET MAUI Windows project using Cake

### DISCUSSION THREAD

**Discussion by PaulAndersonS** [cake-build/cake#3799] [Extension Q&A] [answered]
DotNetCoreAliases.DotNetCoreBuild(Context, @"..\xxx.csproj",
            new DotNetCoreBuildSettings
            {
                NoRestore = true,
                Configuration = configuration,
                MSBuildSettings = new DotNetCoreMSBuildSettings
                {
					TreatAllWarningsAs = MSBuildTreatAllWarningsAs.Default,
                    Verbosity = DotNetCoreVerbosity.Minimal,
                }
            });

Im getting the issue 

C:\Program Files (x86)\Microsoft Visual Studio\Shared\NuGetPackages\microsoft.windows.sdk.buildtools\10.0.22000.194\build\Microsoft.Windows.SDK.BuildTools.props(4,37): error MSB4185: The function "Is64BitProcess" on type "System.Environment" is not available for execution as an MSBuild property function. [D:\Github\xxx.csproj]

Android, ios getting succeeded. Any help

**Comment by JuergenRB** [comment, marked as answer]
Windows App SDK currently doesn't support using the dotnet build command, you have to use the msbuild command instead. - see: https://github.com/microsoft/WindowsAppSDK/issues/1652
There is a possibility to vote for the [feature `dotnet build`](https://portal.productboard.com/winappsdk/1-windows-app-sdk/c/57-support-dotnet-build?utm_medium=social&utm_source=portal_share)

**Reply by PaulAndersonS** [reply]
> Windows App SDK currently doesn't support using the dotnet build command, you have to use the msbuild command instead. - see: [microsoft/WindowsAppSDK#1652](https://github.com/microsoft/WindowsAppSDK/issues/1652) There is a possibility to vote for the [feature `dotnet build`](https://portal.productboard.com/winappsdk/1-windows-app-sdk/c/57-support-dotnet-build?utm_medium=social&utm_source=portal_share)

I have changed as suggested to MSBuild. But I'm not able to generate Appx package. Any idea.

MSBuild(@"..\xxx.sln",GetMSBuildSettings().WithProperty("AppxBundle", "Always").
		                WithProperty("AppxSymbolPackageEnabled", "True").
                                WithProperty("AppxPackageSigningEnabled", "false").
                                WithProperty("AppxPackageDir", @"../AppxPackages").
		                WithProperty("AppxBundlePlatforms", "x64"));

**Comment by nils-a** [comment]
There is also [this answer](https://github.com/cake-build/cake/discussions/3384#discussioncomment-819207), in [discussion 3348: "How to build the MAUI project using Cake plugin?"](https://github.com/cake-build/cake/discussions/3384) that could probably give a hint.
